docs/docs/tutorial/chapter3/saving-data.md

```diff
@@ -195,7 +195,7 @@ export const deleteContact = ({ id }) => {
 
 Pretty simple. You can see here how the `createContact()` function expects the `input` argument and just passes that on to Prisma in the `create()` call.
 
-You can delete `updateContact` and `deleteContact` here if you want, but since there's no longer an assessible GraphQL field for them they can't be used by the client anyway.
+You can delete `updateContact` and `deleteContact` here if you want – since there's no longer an accessible GraphQL field for them they can't be used by the client anyway.
```

Comment:
Ahh yes, that should have been "accessible"!

I want to keep the `, but` version of that sentence. I'm saying that you can delete them if you want, but you don't have to, because they're not accessible anyway.

Reply:
I think i get it now :wink: 

Changed back to keep the "but" in 8fdf043